common/source/docs/common-rangefinder-setup.rst

```diff
@@ -0,0 +1,42 @@
+.. _common-rangefinder-setup:
+
+===========================
+RangeFinders Setup Overview
+===========================
+
+There are many different kinds of rangefinders: Lidar (using laser or infra-red light to measure distance), Sonar (using ultrasonic sound), and Radar (using microwave RF). Some are analog, producing  pulses whose timing represent the distance to an object, others are digital sending data streams over serial to UARTs, or I2C, or even via UAVCAN.
+
+:ref:`RangeFinders (Sonar or Lidar) <common-rangefinder-landingpage>` can be used for :ref:`Object Avoidance <common-object-avoidance-landing-page>` in Copter-3.5 (and higher) in Loiter and AltHold modes and in Rover-3.5 (and higher). As well as altitude sensors for precision landing in Plane and Copter
+
+..  youtube:: y2Kk6nIily0
+    :width: 100%
+
+.. warning::
+   :ref:`common-object-avoidance-landing-page` features are new and should be used with caution.
+   
+
+Up to 10 Rangefinders can be used in the system. :ref:`Object Avoidance <common-object-avoidance-landing-page>` can use a single 360 degree Lidar, or up to 9 unidirectional rangefinders: 8 arranged in a circle covering 45 degree wide segments, plus an upward facing rangefinder. And a downward facing rangefinder used for low altitude height above ground measurements.
+
+Connecting and Configuring the Rangefinder
+==========================================
+
+- follow the instructions for each type of rangefinder described in its linked page on :ref:`common-rangefinder-landingpage`.
+- set the RNGFNDx_ORIENT parameters (i.e. :ref:`RNGFND1_ORIENT <RNGFND1_ORIENT>`, :ref:`RNGFND2_ORIENT <RNGFND2_ORIENT>`, etc.) to specify the direction each range finder is pointing in. 
+[site wiki="copter,rover"]
+  - For Copter and Rover, up to 8 rangefinders may placed around the vehicle to provide 360 degree coverage, or a single 360 degree Lidar. But only one forward facing rangefinder is required for :ref:`Object Avoidance <common-object-avoidance-landing-page>` . Since these may have obstructions in their field of view, so exclusion zones are provided via the PRX_IGN_ANGx and PRX_IGN_WIDx parameters, specifying a direction and width the frame obstruction presents and will be ignored. Up to 6 exclusion sectors can be specified.
```

Comment:
The PRX_IGN_ANG and _WID parameters are only used with the 360 degree lidar (SF40C, RPLidarA2).  The rest don't use it.

Reply:
done, but maybe it should be extended for arrays of unidirectional sensors, two or more?